datarepo-clienttests/src/main/java/scripts/utils/DataRepoUtils.java

```diff
@@ -4,19 +4,7 @@
 import bio.terra.datarepo.api.ResourcesApi;
 import bio.terra.datarepo.client.ApiClient;
 import bio.terra.datarepo.client.ApiException;
-import bio.terra.datarepo.model.BillingProfileModel;
-import bio.terra.datarepo.model.BillingProfileRequestModel;
-import bio.terra.datarepo.model.ConfigEnableModel;
-import bio.terra.datarepo.model.ConfigGroupModel;
-import bio.terra.datarepo.model.ConfigModel;
-import bio.terra.datarepo.model.ConfigParameterModel;
-import bio.terra.datarepo.model.DatasetModel;
-import bio.terra.datarepo.model.DatasetRequestModel;
-import bio.terra.datarepo.model.DatasetSummaryModel;
-import bio.terra.datarepo.model.ErrorModel;
-import bio.terra.datarepo.model.JobModel;
-import bio.terra.datarepo.model.SnapshotRequestModel;
-import bio.terra.datarepo.model.TableModel;
+import bio.terra.datarepo.model.*;
```

Comment:
According to the Google Java style guide, wildcard imports are not allowed. Not sure why the codestyle checker didn't catch this. https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Reply:
Yes, and my settings are supposed to not generate that! Will fix.